Twitter.js

```diff
@@ -95,8 +95,9 @@ function scrape(doc, url) {
 			if (m[0].includes("PM")) {
 				m[0] = m[0].replace("PM", "").replace(/\d+:/, function(matched) {
 					return (parseInt(matched)+12) + ":";
-				});
+				}).trim();
 			}
+			m[0].replace("24:", "00");
```

Comment:
Unfortunately this (and thus line 97 above) isn't quite right. 12:00AM is 00:00, 12:00PM is 12:00

Reply:
Okay, but that is fixable...